src/Chapter3.hs

```diff
@@ -460,6 +483,16 @@ Create a simple enumeration for the meal types (e.g. breakfast). The one who
 comes up with the most number of names wins the challenge. Use your creativity!
 -}
 
+data BreakfastType = American | English | Cantonese  
+data Meal
+    = Breakfast BreakfastType
```

Comment:
I appreciate you spending time on the breakfast type!
English Breakfast is my favourite :yum: :fried_egg: :bacon: :mushroom: :bread: :cheese: